warmup/summary_statistics.py

```diff
@@ -57,21 +65,54 @@ def collect_summary_statistics(data_dictionaries, delta, steady_state):
                 # steady state. However, the last segment may be equivalent to
                 # its adjacent segments, so we first need to know which segments
                 # are steady-state segments.
+                if data_dictionaries[machine]['classifications'][key][p_exec] == 'no steady state':
+                    continue
+                # Capture the last steady state segment for bootstrapping.
```

Comment:
Yeah, I don't think now is the time, hence suggesting an XXX as opposed to immediate modularisation.